titan/features/incar.py

```diff
@@ -130,13 +130,12 @@ def update_agent(self, model: "model.HIVModel"):
                             model.run_random.random()
                             < self.agent.location.params.incar.haart.adherence
                         ):
-                            adherent = True
+                            self.agent.haart.adherent = True  # type: ignore[attr-defined]
```

Comment:
assign haart adherent to the result of the expression in the if

Reply:
So `self.agent.haart.adherent = model.run_random.random() < self.agent.location.params.incar.haart.adherence` right?